package.json

```diff
@@ -65,5 +65,6 @@
   },
   "resolutions.justification": {
     "z-schema": "CVE-2021-3765 in validator. z-schema is used by rush which is a dependency of lage so should not be executed in this repo"
-  }
```

Comment:
Your change shouldn't need to modify the package.json, please revert.

Reply:
Reverted, thank you for the review!